Phishing campaign ux fix (#13419)
demisto/content: Packs/Campaign/ReleaseNotes/2_0_8.md

#### Scripts
##### FindEmailCampaign

> - Split the email summary section to summary and snippets sections and changed the output to be more readable.

Reviewer: ```suggestion
- Improved the readable outputs and split the *Campaign Snippets* layout section from the *Campaign  Summary* section. 
```
